tests/Tests/100__deploy/110__auth_providers_and_rbac/110__rbac.robot

```diff
@@ -15,6 +15,17 @@ Verify Default Access Groups Settings And JupyterLab Notebook Access
     Verify Default Access Groups Settings
     Verify User Can Spawn A Notebook
 
+Verify Empty Group Doesnt Allow Users To Spawn Notebooks
+    [Documentation]   Verify that User is unable to Access Jupyterhub after modifying Access Groups in rhods-groups-config
+    [Tags]    Sanity
+    ...       Tier1
+    ...       ODS-572
+    Run  oc patch configmap rhods-groups-config -n redhat-ods-applications --type="json" -p='[{"op":"replace", "path":"/data/admin_groups", "value":""}]'
```

Comment:
I think lines 23-24 can be replaced by a call to the keyword `Apply Access Groups` Settings in ODS.robot

Reply:
Updated.